config/blobs.yml

```diff
@@ -1,23 +1,23 @@
-haproxy/haproxy-2.2.14.tar.gz:
-  size: 2923727
-  object_id: 2639423c-b8b6-4c52-6fb4-b1ac779eea67
-  sha: sha256:6a9b702f04b07786f3e5878de8172a727acfdfdbc1cefe1c7a438df372f2fb61
+haproxy/haproxy-2.4.0.tar.gz:
```

Comment:
Why not use 2.4.2 instead? Most recent version and has quite some fixes in it.

Reply:
Fixed. Accidentally got the latest release from the wrong list